aries_cloudagent/protocols/actionmenu/v1_0/base_service.py

```diff
@@ -28,19 +29,24 @@ async def get_instance(context: InjectionContext):
 
     @abstractmethod
     async def get_active_menu(
-        self, connection: ConnRecord = None, thread_id: str = None
+        self,
+        profile: Profile,
```

Comment:
Not sure if this is the best way to get access to the profile for the event.notify.

Reply:
The action menu is a bit of a strange case. I'm not sure just yet how we should handle it. Is a profile available easily in the contexts these methods are called from?

Reply:
Yes, in ACA-Py they are only called in handlers

`get_active_menu` is called in MenuRequestHandler
`perform_menu_action` is called in PerformHandler